proto/buf/alpha/registry/v1alpha1/repository.proto

```diff
@@ -46,6 +47,23 @@ message Repository {
   bool deprecated = 8;
   // deprecation_message is the message shown if the repository is deprecated.
   string deprecation_message = 9;
+  // owner_name is the name of the owner of the repository,
+  // either a username or organization name.
+  string owner_name = 10;
+  // description is the user configurable description of the repository.
+  string description = 11;
+  // url is the user configurable URL in the description of the repository,
+  // always included the scheme and will not have a #fragment suffix.
+  string url = 12;
+  // license is the user configurable license of the repository, which should be one of
+  // the identifier defined in https://spdx.org/licenses
+  string license = 13;
```

Comment:
How are custom non-spdx licenses represented here?

Reply:
we can just do "custom", added to the comment

Reply:
I think we should avoid adding our own special value here, if spdx doesn't have a placeholder value but they may in the future I'd rather not guess.

How about we rename this `license_spdx_identifier`, and if it is empty but there is a license url we count that as custom?